core/src/main/java/contacts/core/util/DataRefresh.kt

```diff
@@ -38,4 +38,6 @@ fun <T : CommonDataEntity> T.refresh(contacts: Contacts, cancel: () -> Boolean =
             Include(fields(contacts.customDataRegistry) + Fields.Required.all),
             null, CompoundOrderBy(setOf(Fields.DataId.asc())), 1, 0, cancel
         ).firstOrNull()
-    }
\ No newline at end of file
+    }
+
+// TODO MutableData extension + async
```

Comment:
The changes in this file technically fixes #109 because the exception can no longer occur because `MutableData` instances can no invoke this `refresh` function. However, we still need to implement the extension for `MutableData`, hence this TODO. It will be addressed as part of the complete fix for #109.